lib/components/DynamicFonts/index.ts

```diff
@@ -118,13 +112,13 @@ export default class DynamicFonts {
     }
   }
 
-  private buildFontName(rootUri: string,
+  private buildFontInput(rootUri: string,
```

Comment:
I don't understand this name, shouldn't it be more like 'getFontData' or something?

Reply:
I wanted to reflect the type `GetFontInput`.
I think I'll change `GetFontInput` -> `GetFontOptions` and `buildFontInput` -> `buildFontOptions` (or `getFontOptions` if you prefer).
WDYT?

Reply:
Are they options or required?

Reply:
Good point, mostly required